### ISSUE TITLE
Containerized workflow

### ISSUE THREAD

**Issue by Zlopez** [fedora-infra/the-new-hotness#336]
# Description
Because the-new-hotness is running in OpenShift it will be nice to have similar environment for development. This means using containers instead of Vagrant.

This will be probably challenging because the-new-hotness is connecting to multiple external systems in Fedora infrastructure that need credentials, which are not provided in advance and you need to fill them before starting the-new-hotness service in vagrant.

## Requirements
* Create a container for the-new-hotness
* Create a container for RabbitMQ
* the-new-hotness container should be able to connect to RabbitMQ
* Update contribution guide
* (Optional) Use pytest-docker for integration testing
* (Optional) Add redis container

## How this will make the-new-hotness better
This will make production env and dev env almost one to one and will allow us to reproduce issues we encounter in production much more easily.
